schemas/Product.yaml

```diff
@@ -1,20 +1,154 @@
-required:
+# copyright 2021 Lucid Green, Inc.
+
+required: 
   - id
   - name
-  - strainId
-  - strainName
-  - productCategoryName
-  - productCategoryType
-  - quantityType
-  - publicIngredients
-  - description
-  - isUsed
+  - updated_on
+  - product_type
+  - usage
+  - variety
+  - package_size
 properties:
   id:
     $ref: Id.yaml
+  brand_id:
+    $ref: Id.yaml
+  updated_on:
+    type: string
+    format: date-time
+    description: The date and time when the details of this product were last updated.
   name:
     type: string
     example: Kush OG
+  sku:
+    type: string
+  upc:
+    type: string
+  barcode:
+    type: string
+  product_line:
+    type: string
+  product_type:
+    type: string
+    enum:
+     - "blunt"
```

Comment:
I know this may complicate things, but I think the camelcase thing should apply here too. I am pretty sure that we should keep enums in line with the spec and using camelcase.

Reply:
ok